.ci.yaml

```diff
@@ -3899,27 +3877,6 @@ targets:
       - bin/**
       - .ci.yaml
 
-  - name: Windows gradle_plugin_fat_apk_test
-    bringup: true # Flaky https://github.com/flutter/flutter/issues/101428
-    recipe: devicelab/devicelab_drone
-    presubmit: false
-    timeout: 60
-    properties:
-      caches: >-
-        [
-          {"name":"gradle","path":"gradle"}
-        ]
-      dependencies: >-
-        [
-          {"dependency": "android_sdk", "version": "version:31v8"},
-          {"dependency": "chrome_and_driver", "version": "version:96.2"},
-          {"dependency": "open_jdk", "version": "11"}
-        ]
-      tags: >
-        ["devicelab","hostonly"]
-      task_name: gradle_plugin_fat_apk_test
-    scheduler: luci
```

Comment:
It's sufficient for this test to run on Linux only where there's more capacity. It's unnecessary to run it on macOS/Windows.